Gemfile

```diff
@@ -27,7 +29,11 @@ gem 'bootsnap', '>= 1.4.4', require: false
 
 group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
+  gem 'rubocop', require: false
```

Comment:
it would be better if you provide gems with versions. Go on this [website](https://rubygems.org/gems/rubocop) and search for a gem, for example
```
gem 'rubocop', '~> 1.18', '>= 1.18.4'
```

Reply:
It is important because if someone clones your repo he would have the same gems versions as you do. We will clone your repositories to play with searching for tough bugs :)